karma/flows/test_controllers.coffee

```diff
@@ -765,3 +765,51 @@ describe 'Controllers:', ->
 
         for rule_test in rule_tests
           expect(scope.isRuleComplete(rule_test['rule'])).toBe(rule_test['complete'])
+
+          
+     it 'should generate json quick replies to send', ->
+      loadFavoritesFlow()
+
+      actionset = flowService.flow.action_sets[0]
+      action = actionset.actions[0]
+    
+      json_quick_responses = [{
+                    'title':'Quick reply',
```

Comment:
let's use `label` and payload for these. Leave `content_type` out for  the time being.